### DISCUSSION TITLE
Cannot open some URLs of Hosted Weblate

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [WeblateOrg/weblate#8686] [Q&A] [answered]
Example: https://hosted.weblate.org/translate/gnu-health/health_dentistry/zh_Hans/?checksum=5b96f12c06e46042
Not sure if it is related to my network.

**Comment by nijel** [comment, marked as answer]
It loads just fine for me, maybe it was just some temporary issue...

**Reply by Geeyun-JY3** [reply]
I can open it right now. Quite werid.
